lib/onebox/mixins/git_blob_onebox.rb

```diff
@@ -169,6 +169,12 @@ def raw
             else
               contents = URI.parse(self.raw_template(m)).open(read_timeout: timeout).read
 
+              if contents.encoding == Encoding::BINARY || true
```

Comment:
Did you mean to leave the `|| true`

```suggestion
              if contents.encoding == Encoding::BINARY
```

Reply:
Sorry, that was a left over from debugging. 🤦🏻‍♀️